update peering info from VPC to Vnet for Azure installs (#1226)
astronomer/docs: astro/install-azure.md

For more information about the resources required to run Astro on Azure, see [Azure Resource Reference](resource-reference-azure.md).

### VNet peering prerequisites (Optional)

> If any Azure resources are on a private network, you can choose between two options:

Reviewer: ```suggestion
To give Astro access to any Azure resources on a private network, you can choose between two options:
```

>  1. Create a VNet Peering connection between Astronomer's VNet and the VNets for your broader network. 
>  2. [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)

Reviewer: ```suggestion
 - Create a VNet Peering connection between Astronomer's VNet and the VNets for your broader network. 
 - [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2).
```

>  2. [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)

Reviewer: What does the user need to do for [Virtual Network Integration](https://docs.microsoft.com/en-us/azure/api-management/virtual-network-concepts?tabs=stv2)?

>  If you want to continue with the option #1, please open a ticket within our [Astronomer Support](https://support.astronomer.io/) with following details and support will proceed on configuring the VNet peering:

Reviewer: ```suggestion
 To implement the first option, contact [Astronomer Support](https://support.astronomer.io/) and provide the following details:
```

 - Azure TenantID of the target VNet
 - Subscription ID of the target VNet
 - Region of the target VNet
 - VNet ID of the target VNet
 - ResourceGroup of the target VNet
 - A CIDR block (RFC 1918 IP Space) no smaller than a `/19` range. You must ensure it does not overlap with the Azure VNet(s) that you will be peering with later. The default CIDR range is `172.20.0.0/19`.

>  - ResourceID - The resource ID is similar to the following example: /subscriptions/<Subscription Id>/resourceGroups/myResourceGroupA/providers/Microsoft.Network/virtualNetworks/myVnetA (you can capture the resource ID within the Azure portal by following step #7 under [this link](https://docs.microsoft.com/en-us/azure/virtual-network/create-peering-different-subscriptions#portal). [Azure Portal login](https://portal.azure.com/)

Reviewer: ```suggestion
 - ResourceID, for example: `/subscriptions/<Subscription Id>/resourceGroups/myResourceGroupA/providers/Microsoft.Network/virtualNetworks/myVnetA`. You can find the resource ID in the Azure portal by following step 7 in [Create peering - Azure portal](https://docs.microsoft.com/en-us/azure/virtual-network/create-peering-different-subscriptions#portal).
```

>  **Please Note:** As outline within the Azure docs [here](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering#permissions), the accounts you use to work with virtual network peering must be assigned to the following roles:
>         -  Network Contributor: For a virtual network deployed through Resource Manager.
>         -  Classic Network Contributor: For a virtual network deployed through the classic deployment model.

Reviewer: ```suggestion
From here, Astronomer support will set up the VNet connection to Astro.
```
Is the account information specific to Astro in any way? If they already are working in VNet, I assume they have the correct permissions on their account.
